plugins/ptp_operator/metrics/metrics.go

```diff
@@ -74,12 +75,12 @@ const (
 
 // ExtractMetrics ... extract metrics from ptp logs.
 func (p *PTPEventManager) ExtractMetrics(msg string) {
-	defer func() {
+	/*defer func() {
```

Comment:
delete if code not used.

Reply:
thanks we need this to prevent restarts of the pod